sdk-commons/src/test/java/org/openmrs/maven/plugins/model/SdkStatisticsTest.java

```diff
@@ -37,4 +42,20 @@ public void shouldSetStatsEnabledMode(){
         assertThat(sdkStatistics.getStatsEnabled(), is(true));
     }
 
+    @Test
+    public void testCheckIfOneWeekApart_LessThanOneWeekApart() {
```

Comment:
What i had in mind was tests for the original method `checkIfOneWeekFromLastReport` such that they still pass even after the method internal implementation details are changed.